### ISSUE TITLE
MicrosoftTeams.munki.recipe imports Teams with wrong version number

### ISSUE THREAD

**Issue by opus-nbo** [autopkg/rtrouton-recipes#149]
Microsoft Teams is imported with the version number of the AudioDevice package where it should be the version number of Teams itself.

**Comment by opus-nbo** [comment]
@homebysix Can you assist here? I don't know how to address this, either. I saw you participated in this recipe.

**Comment by homebysix** [comment]
I haven't participated in the Microsoft Teams munki recipe in [this repo](https://github.com/autopkg/rtrouton-recipes/commits/master/MicrosoftTeams/MicrosoftTeams.munki.recipe), but there are a few [recipes in other repos](https://github.com/search?q=org%3Aautopkg+teams+munki&type=code) that you might try to see if they meet your needs.

I'm guessing the recipe needs a `MunkiPkginfoMerger` process, like [this one](https://github.com/autopkg/apizz-recipes/blob/master/Microsoft_Teams/MicrosoftTeams.munki.recipe#L43-L56) has.

**Comment by opus-nbo** [comment]
@grahampugh Can you maybe adapt the changes in the recipe from https://github.com/autopkg/apizz-recipes/blob/master/Microsoft_Teams/MicrosoftTeams.pkg.recipe ?

**Comment by grahampugh** [comment]
I don't have anything to contribute to this issue, sorry. I do not use Munki or write Munki recipes.

**Comment by tpuikkon** [comment]
@opus-nbo I have used this recipe as basis and modified it to our needs. I found this problem also and I think I resolved it in my recipe. I'm not an autopkg pro so there might be easier solution, but this should work (I have tested this).

https://github.com/tpuikkon/misc/blob/main/MicrosoftTeams.munki.recipe I originally added MunkiInstallsItemsCreator to the recipe to collect installs key and now I used it to solve this. So today I added Versioner and MunkiPkginfoMerger to merge that correct version to munki.
